packages/core/src/index.ts

```diff
@@ -1322,7 +1371,17 @@ export abstract class PostHogCore extends PostHogCoreStateless {
   }
 
   /**
-   * * @returns {string} The stored session ID for the current session. This may be an empty string if the client is not yet fully initialized.
+   * Returns the current session_id.
+   *
+   * @remarks
+   * This should only be used for informative purposes.
+   * Any actual internal use case for the session_id should be handled by the sessionManager.
+   *
+   * {@label Session replay}
```

Comment:
this is also used for session duration https://posthog.com/tutorials/session-metrics
not only session replay

> The stored session ID for the current session. This may be an empty string if the client is not yet fully initialized.

the new comment is removing ^ but i dont think we should